docs/apis/core/htmlwriter/index.md

```diff
@@ -0,0 +1,60 @@
+---
+title: HTML Writer API
+tags:
+  - ' API'
+  - ' HTML'
+  - ' DOM'
+---
+
+Moodle has a class called _HTML writer_ which allows you to output basic HTML tags. This is typically used within renderer functions, for example `question/type/*pluginname*/renderer.php`.
+
+:::tip
+Please consider using [templates](../../../guides/templates/index.md), as an alternative to the _HTML writer_
```

Comment:
Can you please put an empty line around the admonitions, that is:
```
:::tip

Content goes here...

:::
```

Some tooling (prettier) will join the lines if we don't.

I need to write a mdlint check for this really.